Dockerfile

```diff
@@ -74,13 +74,15 @@ RUN set -ex; \
             python3-dev \
             apache-arrow-dev \
             py3-pip; \
+        # Upgrade pip to 24.3.1 to match the version in the ha timescaledb-docker image.
+        python3 -m pip install --upgrade pip==24.3.1 --break-system-packages; \
```

Comment:
Without `--break-system-packages`, it complains about being under a managed python venv. Not sure if this is the really idiomatic way to go or rather I should activate the venv somehow from here.

Reply:
Without `--break-system-packages`, it complains about being under a managed python venv. Not sure if this is the really idiomatic way to go or rather I should activate the venv somehow from here.